cleverpush/src/main/java/com/cleverpush/stories/Utils/OnSwipeTouchListener.java

```diff
@@ -0,0 +1,61 @@
+package com.cleverpush.stories.Utils;
+
+import android.content.Context;
+import android.view.GestureDetector;
+import android.view.GestureDetector.SimpleOnGestureListener;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.View.OnTouchListener;
+
+public class OnSwipeTouchListener implements OnTouchListener {
+
+    public final GestureDetector gestureDetector;
+
+    private OnSwipeDown onSwipeDown;
+
+    public OnSwipeTouchListener(Context ctx, View mainView, OnSwipeDown onSwipeDown) {
+        gestureDetector = new GestureDetector(ctx, new GestureListener());
```

Comment:
please use this.gestureDetector to stay unified with onSwipeDown and to make it clear you're assigning a class scope variable.